### ISSUE TITLE
link to lightgbm paper

### ISSUE THREAD

**Issue by dsl10brc** [Microsoft/LightGBM#310]
Can I get link to lightgbm algorithm paper just like we have xgboost algorithm paper (https://arxiv.org/pdf/1603.02754.pdf)?

**Comment by guolinke** [comment]
The paper will release soon. stay tuned.

**Comment by yutkin** [comment]
There is no paper yet?

**Comment by Lucidyan** [comment]
@chivee @guolinke The issue is closed, but I still do not see a link to the official paper. Maybe I missed something?

**Comment by guolinke** [comment]
it is going to release, refer to https://nips.cc/Conferences/2017/AcceptedPapersInitial 

> A Highly Efficient Gradient Boosting Decision Tree

**Comment by henry0312** [comment]
Great!!

**Comment by argman** [comment]
zan!

**Comment by xmpy** [comment]
Hi, is there any update for this paper? thank you :-)

**Comment by henry0312** [comment]
The paper has been published on NIPS, too.
https://papers.nips.cc/paper/6907-lightgbm-a-highly-efficient-gradient-boosting-decision-tree
(there is a supplementary)
